src/mixins/filing-template-mixin.ts

```diff
@@ -596,8 +596,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
 
     // Store share classes and original resolution dates
     this.setShareClasses(cloneDeep(businessSnapshot.shareStructure.shareClasses))
-    this.setResolutionDates([]) // clear new resolution dates
```

Comment:
Don't we still need to clear any new res dates the user entered?

Reply:
I don't think so, if we want to save them for drafts, we would want to preserve whatever they've added so far no?

Reply:
This is `parseBusinessSnapshot` right? I thought it was called to reset/delete the current data? In that case, we would want to clear whatever the user had already entered.

Reply:
ah yes, this is the snapshot.
This is setting the original resolutions to store here. We don't need to set the addedDates to [] as the store already does that.

Reply:
This method is also called to "delete all", so it _should_ delete whatever res dates (and anything else) the user entered.

Here: https://github.com/bcgov/business-edit-ui/blob/b80e6097a40744943840d24bf5adbfdfdb002b5e/src/App.vue#L501

Reply:
🤔 I don't quite understand. We can't delete previous existing Resolution Dates?
Maybe i'm misunderstanding. 

Are you saying we should preserve setting the new resolutions to `[]` when we parse the snapshot?
Only concern is we might delete dates from a draft, but i can look into it!

Reply:
I am saying that when we "delete all", which calls parseBusinessSnapshot(), we need to delete any data the user has entered, which in this case is the new resolutions dates.

Reply:
Would it make more sense to delete the user entered data outside of `parseBusinessSnapshot()` since we also use that when starting a new Alteration and resuming a Draft Alteration?

Reply:
Yes, I think it would make more sense, but there's other data in parseBusinessSnapshot() that is also "cleared" and would have to be moved.
🤷

Reply:
kk, i'll have a look into both scenarios asap :)

Reply:
Added this back in as it doesn't seem to conflict with the drafts 👍 

I was having issues